### DISCUSSION TITLE
Mesh elements error

### DISCUSSION THREAD

**Discussion by ap264** [neperfepx/neper#547] [Q&A] [answered]
I meshed my model and formated into inp extension for exporting it into abaqus. The abaqus input file, when opened in notepad, indicates a CPE3 type element was used to mesh the model, which is a plain strain mesh element. I would like to know if it is possible for me to change the type of element from plain strain to plain stress. for eg CPS3 or CPS4 or CPS4I or CPS8R. 

The command used to generate the mesh is given below
neper -M mytess.tess -dim 2 -rcl 0.6 -meshface "domtype!=2" -nset none -faset none -format inp,msh

Attached is the text from notepad

*Element, type=CPE3
1, 7, 1687, 308
2, 7, 302, 1687
3, 1687, 1689, 1688
4, 1688, 1690, 1687
5, 1687, 1690, 308
6, 1691, 1693, 1692
7, 1692, 1693, 1688
8, 1692, 306, 1691
9, 286, 287, 2
10, 1694, 1696, 1695
11, 1696, 1697, 1695
12, 314, 1696, 1694
13, 1698, 1693, 1691
14, 1698, 1691, 295
15, 1699, 1696, 1700
16, 1696, 313, 1700
17, 1697, 1696, 1699
18, 1697, 1699, 1701
19, 1702, 1704, 1703
20, 1704, 1705, 1703

**Comment by rquey** [comment, marked as answer]
For now, the easiest way is by modifying the file:
```
sed -i "s/\<CPE3\>/CPS3/g" file.txt
```
